playbooks/reboot_cluster.yml

```diff
@@ -14,6 +14,14 @@
               delay=15
     delegate_to: "{{ groups['controller']|first }}"
 
+  - name: Ensure Galera is up before rebooting next node
+    wait_for: port=4567
+
+  - name: Ensure mysql is running
+    service:
+      name: mysql
+      state: started
```

Comment:
Can you switch these tests to just checking cluster health status, like in https://github.com/blueboxgroup/ursula/blob/master/roles/percona-server/tasks/main.yml#L67-L75